crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -200,16 +200,9 @@ end
 #! Where:
 #! - ACCOUNT_COMMITMENT is the commitment of the account data.
 export.compute_current_commitment
```

Comment:
Should we add a comment somewhere here about why we don't need a separate "dirty" flag for the entire account? 

I think the reasoning is that, assuming storage commitment is current, computing account commitment is relatively cheap - just 2 permutations of the hash function and probably a total of 20 - 30 cycles. Adding a separate caching mechanisms to avoid this is probably not worth it.